monai/transforms/io/array.py

```diff
@@ -112,6 +112,8 @@ def __init__(
         dtype: DtypeLike = np.float32,
         ensure_channel_first: bool = False,
         simple_keys: bool = False,
+        pattern: Optional[str] = None,
```

Comment:
For the args in `LoadImage`, it's not easy to guess the meaning based on class name (different from `ensure_torch_and_prune_meta`), I would suggest to use `prune_meta_pattern` and `prune_meta_sep`.

What do you think?

Thanks.